distrax/_src/distributions/gamma_test.py

```diff
@@ -0,0 +1,215 @@
+      ('1d dist, 1d value, edge case', (2.1, 1), 200),
+  )
+  def test_log_prob(self, distr_params, value):
+    distr_params = (np.asarray(distr_params[0], dtype=np.float32),
+                    np.asarray(distr_params[1], dtype=np.float32))
+    value = np.asarray(value, dtype=np.float32)
+    super()._test_attribute(
+        attribute_string='log_prob',
+        dist_args=distr_params,
+        call_args=(value,),
+        assertion_fn=self.assertion_fn)
+
+  @chex.all_variants
+  @parameterized.named_parameters(
+      ('1d dist, 1d value', (3.1, 1), 1),
+      ('1d dist, 2d value', (0.5, 0.1), np.array([1, 2])),
+      ('1d dist, 2d value as list', (0.5, 0.1), [1, 2]),
+      ('2d dist, 1d value', (0.5 + np.zeros(2), 0.3 * np.ones(2)), 1),
+      ('2d broadcasted dist, 1d value', (0.4 + np.zeros(2), 0.8), 1),
+      ('2d dist, 2d value', ([0.1, -0.5], 0.9 * np.ones(2)), np.array([1, 2])),
+      ('1d dist, 1d value, edge case', (2.1, 1), 200),
+  )
+  def test_prob(self, distr_params, value):
+    distr_params = (np.asarray(distr_params[0], dtype=np.float32),
+                    np.asarray(distr_params[1], dtype=np.float32))
+    value = np.asarray(value, dtype=np.float32)
+    super()._test_attribute(
+        attribute_string='prob',
+        dist_args=distr_params,
+        call_args=(value,),
+        assertion_fn=self.assertion_fn)
+
+  @chex.all_variants
+  @parameterized.named_parameters(
+      ('1d dist, 1d value', (3.1, 1), 1),
+      ('1d dist, 2d value', (0.5, 0.1), np.array([1, 2])),
+      ('1d dist, 2d value as list', (0.5, 0.1), [1, 2]),
+      ('2d dist, 1d value', (0.5 + np.zeros(2), 0.3 * np.ones(2)), 1),
+      ('2d broadcasted dist, 1d value', (0.4 + np.zeros(2), 0.8), 1),
+      ('2d dist, 2d value', ([0.1, -0.5], 0.9 * np.ones(2)), np.array([1, 2])),
+      ('1d dist, 1d value, edge case', (2.1, 1), 200),
+  )
+  def test_cdf(self, distr_params, value):
+    distr_params = (np.asarray(distr_params[0], dtype=np.float32),
+                    np.asarray(distr_params[1], dtype=np.float32))
+    value = np.asarray(value, dtype=np.float32)
+    super()._test_attribute(
+        attribute_string='cdf',
+        dist_args=distr_params,
+        call_args=(value,),
+        assertion_fn=self.assertion_fn)
+
+  @chex.all_variants(with_pmap=False)
+  @parameterized.named_parameters(
+      ('entropy', ([0., 1.3, -0.5], [0.5, 1.3, 1.5]), 'entropy'),
+      ('entropy broadcasted concentration', (0.5, [0.5, 1.3, 1.5]), 'entropy'),
+      ('entropy broadcasted rate', ([0.1, 1.3, -0.5], 0.8), 'entropy'),
+      ('mean', ([0.1, 1.3, -0.5], [0.5, 1.3, 1.5]), 'mean'),
+      ('mean broadcasted concentration', (0.5, [0.5, 1.3, 1.5]), 'mean'),
+      ('mean broadcasted rate', ([0.1, 1.3, -0.5], 0.8), 'mean'),
+      ('variance', ([0.1, 1.3, -0.5], [0.5, 1.3, 1.5]), 'variance'),
+      ('variance broadcasted concentration', (0.5, [0.5, 1.3, 1.5]), 'variance'),
```

Comment:
nit: line too long, please remove one character (otherwise it makes pylint complain)